peps/pep-0772.rst

```diff
@@ -184,201 +184,191 @@ making process that governs the technical projects under the PyPA umbrella.
-company and its subsidiaries, or another incorporated entity such as a
-non-profit or educational institution with its own mission and goals. PSF
-members are explicitly exempt from this restriction, but *not* PSF directors
-or officers.
+No more than two Packaging Council member may be employed by or significantly affiliated with the same
+entity. An entity is a company and its subsidiaries, or another incorporated entity such as a non-profit or
+educational institution with its own mission and goals. Significant affiliation includes, at the least,
+receiving wages for current work, officer/director roles, and 25% or more ownership stake.  PSF membership at
+any member class (as defined by Article IV of the PSF bylaws) does not count as an affiliation for this
+purpose.
 
-While we trust council members to act in the best interests of Python rather
-than themselves or their employers, the mere appearance of any one company
-dominating Python development could itself be harmful and erode trust.
+While we expect and trust Packaging Council members to act in the best interests of Python rather than
+themselves or their affiliations, the mere appearance of any one such organization dominating Python
+development could itself be harmful and erode trust.
 
-PSF staff members are not permitted to be members of the Packaging Council.
+PSF staff members are not permitted to serve as members of the Packaging Council.
 
-In a council election, if more than two of the top five vote-getters work for
-the same employer, then whichever of them ranked lowest is disqualified and the
-6th-ranking candidate moves up into 5th place; this is repeated until a valid
-council is formed.
+Currently serving Steering Council members are not permitted to concurrently serve as members of the Packaging
+Council.
 
-During a council term, if changing circumstances cause this rule to be broken
-(for instance, due to a council member changing employment), then one or more
-council members must resign to remedy the issue, and the resulting vacancies
-can then be filled as normal.
+In a Council election, if more than two of the top five vote-getters work for the same employer, then only the
+top two such vote-getters are elected and the others are disqualified, with the remaining vote-getters
+elevated in the vote rank.  This process is repeated until a valid Packaging Council is formed.  If after this
+process a full Council cannot be formed, disqualified vote-getters are re-qualified in the rank order of their
+vote tally until a full Council can be formed.
 
-.. _voting-member:
+During a Packaging Council term, if changing circumstances cause this rule to be broken (for instance, due to
+a Council member changing employment), then one or more Council members must resign to remedy the issue, and
+the resulting vacancies can then be filled as `normal <vacancy>`_.
 
---------------
-Voting members
---------------
+.. _electors:
```

Comment:
You have this twice, they need to be unique.

Reply:
Already fixed! 😄